### PR TITLE
Allow users to edit class

### PR DESCRIPTION
**Pull request by jenmccarthy**
Adds a cohort (class) field into the edit registration view for updating. Tests the integration of that feature in specs.

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -24,6 +24,11 @@
       <%= f.email_field :email, autofocus: true %>
     </div>
 
+    <div>
+      <%= f.label :cohort_id, "Class" %>
+      <%= f.select(:cohort_id, Cohort.all.collect {|c| [ c.description, c.id ] }, :include_blank => true) %>
```
Comment (jenmccarthy) [inline app/views/devise/registrations/edit.html.erb]: no blank on edit

File: spec/features/edit_profile_page_spec.rb [hunk 2]
```diff
@@ -22,4 +22,16 @@
       expect(page).to have_content "Current password is invalid"
     end
   end
+
```
Comment (jenmccarthy) [inline spec/features/edit_profile_page_spec.rb]: meh no need for this test

**Comment by michaelrkn** [comment]
Seems like this didn't actually get merged. Can you merge master into this branch, and then we can merge it into master? Thanks!

**Comment by jenmccarthy** [comment]
The Travis CI build is failing on credit card specs--I know there is one that is faulty and fails periodically, is this the case with both these?

**Comment by michaelrkn** [comment]
Yeah those failing tests are the sporadic ones. You can see that the build succeeded the second time. You can also restart the build in Travis if you want to double-check. This is good to merge.

**Pull request merged**